js/index.js

```diff
@@ -9,12 +9,53 @@ copyright.innerHTML = `Diana Carachure ${thisYear}`;
 
 footer.appendChild(copyright);
 
-const skills = ['JavaScript', 'HTML', 'CSS'];
+const skills = ['JavaScript', 'HTML', 'CSS', 'Access', 'Excel','Data Entry', 'Apricot by Social Solutions'];
 const skillsSection = document.getElementById('skills');
 const skillsList = skillsSection.querySelector('ul'); //replace document with skillsSection -only "looking" in that section
 
-for (let i = 0; i < skills; i++){
-    const skill = document.createElement('li');
-    skill.innerHTML = `${skills}`;
+for (let i = 0; i < skills.length; i++){
+    let skill = document.createElement('li');
+    skill.innerText = skills[i];
     skillsList.appendChild(skill);
-};
\ No newline at end of file
+};
+
+//code below will handle the messages submitted in the form section of the page
+let messageForm = document.getElementsByName("leave_message");
+messageForm.addEventListener('submit', (e) => {
+    e.preventDefault();
+
+//retrieving the value from each event
+let name = e.target.name;
+let email = e.target.email;
+let message = e.target.message;
+
+console.log(name);
+console.log(email);
+console.log(message);
+
+//is not logging values in console when it should be
+
+//following section will display messages in a list
+
+let messageSection = document.getElementById("messages");
+let messageList = messageSection.querySelector("ul");
+let newMessage = document.createElement("li");
+
+newMessage.innerHTML = `<a href= "mailto: ${email}"> ${name} </a> wrote: <span> ${message} </span>`;
+
+//creates a new buttom named remove
+
+let removeButton = document.createElement("button");
+removeButton.innerText = "remove";
+removeButton.type = "button";
+removeButton.addEventListener('click', () => {
+    let entry = removeButton.parentNode;
+    entry.remove();
+});
+
+newMessage.appendChild(removeButton);
+
+messageList.append(messageList);
```

Comment:
```suggestion
messageList.append(newMessage);
```
You don't want to append the list to itself 😄